Demos/04-add-msgraph/README.md

```diff
@@ -9,10 +9,17 @@ The last step is to incorporate the Microsoft Graph into the application. For th
     1. Add the following code to the `dependencies` section, immediately after the previously added dependencies:
 
         ```gradle
-        implementation 'com.microsoft.graph:msgraph-sdk-android:1.3.2'
+        implementation 'com.microsoft.graph:microsoft-graph:0.1.+'
         ```
 
     1. Sync the dependencies with the project by selecting **File > Sync Project with Gradle Files**.
+        
+        if cannot reslove the package, add the following code to the project build.gradle file 'allprojects' section:
```

Comment:
Put this in a callout (with the `>` character). Also wrap the code in a fenced code block.

Reply:
Updated it. Thanks.